### ISSUE TITLE
footer - 'need to get in touch' changes

### ISSUE THREAD

**Issue by Dan-KeduTesting** [Buckinghamshire-Digital-Service/hr-service-desk-alpha#91]
changes needed:

- should be link to email 
- smaller test 
- email icon to right of text
- can add aria label of "Get in touch by email" to link

also email icon should have aria-hidden=true

![footer_-_'need_to_get_in_touch'_changes](https://user-images.githubusercontent.com/2722522/65418779-1628a800-ddf5-11e9-8c41-7356fb9bc000.png)
